test/functional/wallet_balance.py

```diff
@@ -30,10 +30,15 @@ def create_transactions(node, address, amt, fees):
         ins_total += utxo['amount']
         if ins_total + max(fees) > amt:
```

Comment:
Shouldn't this also say?
```suggestion
        if ins_total >= amt + max(fees):
```

Reply:
ah yeah I first had that but then rebased on master and noticed that someone already added that `max(fees)` check. I will edit it to be `>=` instead.

Reply:
changed